capture-sdk/sdk/src/main/res/values/styles.xml

```diff
@@ -202,7 +203,7 @@
     </style>
 
     <style name="Root.GiniCaptureTheme.Help.Item.TextStyle" parent="TextAppearance.AppCompat">
```

Comment:
[Clarification] This style is not used anymore, right? Just asking because I see you modified it.